fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/AggregatedMaterializedViewPushDownRewriter.java

```diff
@@ -97,41 +98,6 @@ public OptExpression doRewrite(MvRewriteContext mvContext) {
         }
```

Comment:
suggest to avoid time-consuming cloning if the rule fails to transform the plan

Reply:
fixed.